### DISCUSSION TITLE
Niche selection

### DISCUSSION THREAD

**Discussion by natalie-ne** [dailydotdev/daily#1196] [Feature Requests]
I love daily.dev and been updated with all the topic I'm intersted in.
I understand the focus is on developments/tech articles/blogs. 
I suggest adding niche options:
1. Marketing, social media, digital creator etc...
2. Identify users who tries different keyword searches, calc the "hot topics" and add more content (aim your client needs)

**Comment by idoshamun** [comment]
Hi @natalie-ne,

Our target audience is strictly developers and we're very much focused on that.
Expanding further is out of scope for us, we prefer to dive deeper and improve our offerings for developers.
Thanks a lot for the suggestion! 🙏
